.github/workflows/trivy.yml

```diff
@@ -1,46 +1,33 @@
+# Build the Docker image and scan it with Trivy
 name: dsp-appsec-trivy
-on: [pull_request]
+on:
+  workflow_dispatch: {}
```

Comment:
Just curious if we want to leave in the `workflow_dispatch` option here or if it was added solely for testing?  I don't mind leaving it in.

Reply:
I think it's worth keeping this so we can test and run our actions whenever needed. For example, I could imagine running this manually if we suspect that a new vulnerability affects the base image.